### DISCUSSION TITLE
请问如何卸载DLL？注入之后dll列表里面没有wxhelper.dll

### DISCUSSION THREAD

**Discussion by cocktail18** [ttttupup/wxhelper#295] [Q&A]
使用工具`https://github.com/nefarius/Injector`注入成功之后，dll列表里面没有wxhelper.dll
反而多了下面几个dll
MSVCP140.dll
VCRUNTIME140.dll
VCRUNTIME140_1.dll

卸载dll的时候要怎么卸载呢？

**Comment by ttttupup** [comment]
隐藏了，暂时没考虑卸载
